application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -95,6 +97,16 @@ export class ViewSearchInput
                     }
                 }),
         );
+        this.env().subscriber.register(
+            this.session.search
+                .state()
+                .subjects.get()
+                .active.subscribe((filter: IFilter | undefined) => {
+                    if (filter !== undefined) {
+                        this.active = new ActiveSearch(this.session.search, filter);
+                    }
```

Comment:
just don't know (didn't test) - do we need here `this.detectChanges()`?

Reply:
I've tested it just now and it works without